packages/zoe/src/zoeService/zoeStorageManager.js

```diff
@@ -182,113 +215,207 @@ export const makeZoeStorageManager = (
+          getRoot() {
+            return root;
+          },
+          getWithdrawFacet() {
+            const { facets } = this;
+            return facets.withdrawFacet;
+          },
+          getAdminNode() {
+            return adminNode;
+          },
+        },
+        // Goes to the Zoe seat, which isn't restricted in how much to withdraw
+        withdrawFacet: {
+          withdrawPayments(amounts) {
+            return escrowStorage.withdrawPayments(amounts);
+          },
+        },
+      },
+    );
+    return makeInstanceStorageManager().instanceStorageManager;
   };
 
-  return {
-    makeZoeInstanceStorageManager,
-    getAssetKindByBrand: issuerStorage.getAssetKindByBrand,
-    depositPayments: escrowStorage.depositPayments,
-    invitationIssuer,
-    installBundle,
-    installBundleID,
-    getBundleIDFromInstallation,
-    getPublicFacet,
-    getBrands,
-    getIssuers,
-    getOfferFilter,
-    getTerms,
-    getInstallationForInstance,
-    getInstanceAdmin,
-    unwrapInstallation,
-    getProposalShapeForInvitation,
+  const installBundle = bundleID => {
+    return installationStorage.installBundle(bundleID);
   };
+
+  const getInvitationIssuer = () => invitationIssuer;
+
+  const makeStorageManager = vivifyFarClassKit(
+    zoeBaggage,
+    'ZoeStorageManager',
+    ZoeStorageManagerIKit,
+    instanceAdmins => ({ instanceAdmins }),
+    {
+      zoeServiceDataAccess: {
+        getInvitationIssuer,
+        getBundleIDFromInstallation(allegedInstallation) {
+          return installationStorage.getBundleIDFromInstallation(
+            allegedInstallation,
+          );
+        },
+        getPublicFacet(instance) {
+          const { state } = this;
+          return state.instanceAdmins.getPublicFacet(instance);
+        },
+        getBrands(instance) {
+          const { state } = this;
+          return state.instanceAdmins.getBrands(instance);
+        },
+        getIssuers(instance) {
+          const { state } = this;
+          return state.instanceAdmins.getIssuers(instance);
+        },
+        getOfferFilter(instance) {
+          const { state } = this;
+          return state.instanceAdmins.getOfferFilter(instance);
+        },
+        setOfferFilter(instance, filters) {
+          const { state } = this;
+          state.instanceAdmins.setOfferFilter(instance, filters);
+        },
+        getTerms(instance) {
+          const { state } = this;
+          return state.instanceAdmins.getTerms(instance);
+        },
+        getInstallationForInstance(instance) {
+          const { state } = this;
+          return state.instanceAdmins.getInstallationForInstance(instance);
+        },
+        getProposalShapeForInvitation,
+        installBundle,
+        installBundleID(bundleID) {
+          return installationStorage.installBundleID(bundleID);
+        },
+      },
+      makeOfferAccess: {
+        getAssetKindByBrand: issuerStorage.getAssetKindByBrand /* o */,
+        installBundle,
```

Comment:
Aha! This `installBundle` method is the exact same as the `installBundleMethod` of the `zoeServiceDataAccess` facet. They should then have the same parameter and return guards.

Reply:
Looks done. Resolving.